integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/transport/test/ESBJAVA1832MessageInjectorTestCase.java

```diff
@@ -41,11 +44,11 @@
     @BeforeClass(alwaysRun = true)
     protected void init() throws Exception {
         super.init();
-        OMElement msgProessor = esbUtils.loadResource("/artifacts/ESB/jms/transport/msgInjection/msg_store.xml");
-        OMElement task = esbUtils.loadResource("/artifacts/ESB/jms/transport/msgInjection/msg_injecting_task.xml");
         consumer = new JMSQueueMessageConsumer(JMSBrokerConfigurationProvider.getInstance().getBrokerConfiguration());
-        updateESBConfiguration(JMSEndpointManager.setConfigurations(msgProessor));
-        esbUtils.addScheduleTask(contextUrls.getBackEndUrl(), getSessionCookie(), task);
+        OMElement omElement = AXIOMUtil.stringToOM(FileUtils.readFileToString(new File(getESBResourceLocation()
+                + File.separator + "jms" + File.separator + "transport" + File.separator + "msgInjection" + File.separator
+                + "ESBJAVA1832MessageInjectorTestTask.xml")));
+        Utils.deploySynapseConfiguration(omElement, "ESBJAVA1832MessageInjectorTestTask", "tasks", true);
```

Comment:
are we doing this restart just to avoid logs getting printed from schedule tasks ?.  Or is the configuration updated dynamically

Reply:
We tests for the logs that are coming with the task execution. So, we need to restart the server to get logs at the testcase.